{
  "metadata": {
    "timestamp": "2025-02-21T08:14:14Z"
  },
  "data": [
    {
      "cve": {
        "id": "CVE-2024-4032",
        "description": "The “ipaddress” module contained incorrect information about whether certain IPv4 and IPv6 addresses were designated as “globally reachable” or “private”. This affected the is_private and is_global properties of the ipaddress.IPv4Address, ipaddress.IPv4Network, ipaddress.IPv6Address, and ipaddress.IPv6Network classes, where values wouldn’t be returned in accordance with the latest information from the IANA Special-Purpose Address Registries.\n\nCPython 3.12.4 and 3.13.0a6 contain updated information from these registries and thus have the intended behavior.",
        "metadata": {
          "published_date": "2024-06-17T15:15:52.517",
          "last_modified_date": "2024-11-21T09:42:03.990",
          "confidence_level": "low",
          "severity": "high"
        },
        "impact": {
          "cisa_kev": false,
          "reported_exploited": false,
          "exploit_maturity": "none",
          "automatable": true
        },
        "counts": {
          "public_exploit_count": 0
        },
        "epss": {
          "score": 0.00046,
          "percentile": 0.19561
        },
        "metrics": [
          {
            "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
            "type": "Secondary",
            "cvss_version": "3.1",
            "vector_string": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
            "attack_vector": "NETWORK",
            "base_score": 7.5
          }
        ],
        "timeline": {
          "references": [],
          "repositories": []
        }
      }
    }
  ]
}